stable/shared/file-assertions.yaml

```diff
@@ -0,0 +1,105 @@
+ceph-mon:
+  files:
+    - path: /etc/ceph/ceph.conf
+      owner: root
+      group: root
+      mode: '777'
```

Comment:
Why are we checking that ceph.conf is executable ?

Reply:
oh hang on , is this a symlink?

Reply:
that's an `install-alternative` 0 the /var/lib/juju/$whatever` path is to the real file